meson.build

```diff
@@ -1396,15 +1396,16 @@ pam_paths = [
     '/usr',
     '/usr/local',
 ]
-
-if host_os == 'sunos'
-    pam_dir += '/etc/pam.conf'
+message('atalk_libname =', atalk_libname)
+if run_command('uname', '-a', check: true).stdout().strip().contains('omnios')
```

Comment:
Any idea how to make this more portable? The are others in the OpenSolaris family, for instance, which we don't know for sure yet how they function, e.g. SmartOS, OpenIndiana... And there might be particular revisions of said OSes which are generally compatible but has the old PAM structure.

Reply:
For instance keep "if sunos" as a fallback branch in the decision tree, in the case no "pam.d" dir is found.

Besides, autotools throws this warning in the configure summary when no "pam.d" is found:

```
configure: WARNING:  PAM support was configured for your system, but the netatalk PAM configuration file
configure: WARNING:  cannot be installed. Please install the config/netatalk.pamd file manually.
configure: WARNING:  If you're running Solaris or BSD you'll have to edit /etc/pam.conf to get PAM working.
configure: WARNING:  You can also re-run configure and specify --without-pam to disable PAM support.
```

I don't know if the latter point is still valid for *all* Solaris and BSD variants these days, however. They are becoming more POSIX/GNU-ish over time. :)

And finally, I don't think we have an equivalent to `--without-pam` in the Meson build system now, do we?

Reply:
Added option to disable PAM support. If pam.d dir is not found and the host_os is sunos (all solaris-based systems) then a message advises that /etc/pam.conf needs to be edited to enable PAM support